tox.ini

```
@@ -34,6 +34,9 @@ commands =
     coverage run -p -m pytest --durations=20 {posargs:tests}
     coverage combine
     coverage xml
+deps =
+    torch10: torch>=1.10.0,<1.11.0
```

Can we use `~=` instead?
```suggestion
    torch10: torch~=1.10.0
```